RELEASES.md

```diff
@@ -0,0 +1,44 @@
+# CITGM Release Process
+
+This document describes the technical aspects of the citgm release process. The intended audience is those who have been authorized to create official releases for citgm, hosted on npm.
+
+## Who can make a release?
+
+Any member of the @nodejs/citgm team can make a release.
+
+## How to create a release
+
+1. Ensure your branch is up to date with upstream
+
+```bash
+$ git remote update -p
+$ git rebase upstream/master
```

Comment:
What branch is this? I'd assume it should always be master, in which case I'd normally do:

```bash
git checkout master
git fetch upstream # or git fetch --all
git reset --hard upstream/master
```

to be clear about what branch you should be on.

---

After a quick look I'm not sure what the difference is between `git remote update -p` and `git fetch --all -p` (or why you'd bother with the `--prune`. I'm more familiar with `fetch`, but it's NBD either way.